### DISCUSSION TITLE
Indonesia - How Long Should You Be on Github Sponsor Waitlist?

### DISCUSSION THREAD

**Discussion by andae** [community/community#18251] [Other Feature Feedback, Questions, & Ideas]
It's been about 3 years since Github Sponsor was launched, and until now developers and the open source community in Indonesia are still on the waiting list for the program. Listen @github You should soon launch the Github Sponsor program available in Indonesia, to support the open source community in Indonesia.

**Comment by inamira** [comment]
Yap, seems so long on the waiting list. Our community at [UmentorJs](https://umentor.js.org/) are also still waiting for Github Sponsor to be available at [Indonesia](https://github.com/GitIndonesia/awesome-indonesia-repo), hopefully we can catch up like [Brazil](https://github.blog/2022-06-01-github-sponsors-launches-in-brazil/) and [India](https://github.blog/2022-05-23-github-sponsors-launches-in-india/). So let's call @github @github-staff @jlord to listen!!

**Comment by andae** [comment]
Hi @inamira 'You are absolutely right, wish the [Github Sponsor](https://github.com/sponsors) was available in Indonesia like [Brazil](https://github.blog/2022-06-01-github-sponsors-launches-in-brazil/) and [India](https://github.blog/2022-05-23-github-sponsors-launches-in-india/).

**Comment by MrKrishnaAgarwal** [comment]
[Join the waitlist](https://github.com/login?return_to=%2Fsponsors%2Faccounts) to receive updates when they expand it in Indonesia.
if you will join the waitlist they will do it fast as more people will be applying.
@andae & @inamira

**Comment by sinkaroid** [comment]
Yes absolutely you must wait for long time if you were indonesian, But have you tried to [contact](https://support.github.com/request/account?tags=dotcom-footer%2Chubberfy_account) them? In my case it's not even a year to get approved
